cypress/integration/pages/storyPage/testsForAMPOnly.js

```diff
@@ -0,0 +1,38 @@
+import config from '../../../support/config/services';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRunForAMPOnly = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRunForAMPOnly to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing feastures that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfigForAMPOnly = ({
+  service,
+  pageType,
+}) =>
+  describe(`testsThatFollowSmokeTestConfigForAMPOnly for ${service} ${pageType}`, () => {
```

Comment:
Test is also a duplicate https://github.com/bbc/simorgh/blob/f53a329e6f227c379d868886b76745d06a116686/cypress/integration/pages/photoGalleryPage/testsForAMPOnly.js#L14 So it can be combined in https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllAMPPages.js